tests/unit/admin/test-class-wc-payments-admin.php

```diff
@@ -92,16 +92,11 @@ public function test_it_does_not_render_settings_badge( $is_upe_settings_preview
 
 	public function test_it_renders_payments_badge_if_stripe_is_not_connected() {
 		global $menu;
-
-		// The badge is temporarily not shown at all. See
-		// class-wc-payments-admin.php line 227. This should be removed when
-		// implementing https://github.com/automattic/woocommerce-payments/issues/2071.
-		static::markTestSkipped( 'Not yet implemented.' );
-
 		$this->mock_current_user_is_admin();
 
 		// Make sure we render the menu with submenu items.
 		$this->mock_account->method( 'try_is_stripe_connected' )->willReturn( false );
+		update_option( 'wcpay_activation_timestamp', time() - ( 86400 * 7 ) );
```

Comment:
It looks like this test is exactly the same as the newly-introduced `test_it_renders_payments_badge_if_activation_date_is_older_than_7_days`. Let's just keep one and rename it to mention both things (Stripe not being connected and activation ≥ 7 days).

Reply:
Updated in https://github.com/Automattic/woocommerce-payments/pull/2313/commits/be91a5001568b7f6487837a14d7b804b2a3436c8